tox.ini

```diff
@@ -0,0 +1,24 @@
+[tox]
+minversion = 2.3.1
+envlist = py27dj19
+skipsdist = True
+
+[testenv]
+usedevelop = True
+install_command = pip install -c{env:UPPER_CONSTRAINTS_FILE:https://git.openstack.org/cgit/openstack/requirements/plain/upper-constraints.txt} {opts} {packages}
+setenv = VIRTUAL_ENV={envdir}
+deps = -r{toxinidir}/requirements.txt
+       -r{toxinidir}/dev_requirements.txt
+commands =
+  find /opt/dev/atmosphere -type f -name "*.pyc" -delete
+whitelist_externals=
+  find
+  bash
+
+[testenv:py27dj19]
+commands =
+  pip install -U django>=1.9,<1.10
```

Comment:
This is the standard way to do it. Potentially we need to test with different version of Django as the project grows.